src/Psalm/Internal/Cli/Psalm.php

```diff
@@ -638,8 +638,8 @@ private static function initProviders(array $options, Config $config, string $cu
     }
 
     /**
-     * @param array{"set-baseline": string} $options
-     * @return array<string,array<string,strict-array{o:int, s: list<string>}>>
+     * @param array{"set-baseline": string}&array $options
```

Comment:
Shouldn't it use the new `array{"set-baseline": string, ...}` syntax?

Reply:
I'll fix this separately, and remove all remaining uses of this (while keeping it for BC)